pages/tidal tools/analyze_database.md

```diff
@@ -61,6 +61,8 @@ For example, in Oracle databases, the Data Dictionary and AWR repository tables
 
 You can can create a user with the [script and set of permissions defined in this script](/oracle_user.sql). You should provide a secure password on the first line.
 
+If you are using a CDB database you will also need to create a second user to access to CDB, you can use [this script](/oracle_cdb_user.sql) and also provide a secure password at the top.
```

Comment:
“Click here” links will work awful in PDF

Reply:
Yea agreed. How can we handle this better?

Given the size and length of the script file, its too large to put on the page directly. My thought is, we will have to adjust this content specifically for the PDF output vs the website.

Reply:
https://github.com/tidalmigrations/tidalmigrations.github.io/blob/master/oracle_cdb_user.sql is just a seven lines long, I believe it's OK to embed it directly. WDYT?